constraint/pkg/apis/constraints/apis.go

```diff
@@ -45,3 +64,101 @@ func GetEnforcementAction(constraint *unstructured.Unstructured) (string, error)
 
 	return action, nil
 }
+
+func IsEnforcementActionScoped(action string) bool {
+	return strings.EqualFold(action, EnforcementActionScoped)
```

Comment:
@maxsmythe were you meaning to refer to some other codeblock? I tested the switch in the code you shared. I do not think it checks for case sensitivity.

Reply:
@JaydipGabani From the code Max linked, it means we should preserve the case sensitivity of enforcementAction instead of converting it to all lowercase. Currently, only these are expectedhttps://github.com/open-policy-agent/gatekeeper/blob/master/pkg/util/enforcement_action.go#L14-L18 so unless users pass in `deny`, `dryrun`, `warn`, `scoped` (new one that we should add to that list), we should set it as `unrecognized`.

Reply:
Checking for case sensitve matching for actions now.